tests/unit/moduleapi/reply.tcl

```diff
@@ -28,8 +28,19 @@ start_server {tags {"modules"}} {
             assert_equal 3.141 [r rw.double 3.141]
         }
 
+        set ld 3.14159265359
+        if {$::valgrind} { set ld 1.00000000005 } ;# valgrind uses 64 bit long double
         test {RM_ReplyWithLongDouble: a float reply} {
```

Comment:
`test {RM_ReplyWithLongDouble: a float reply - resp$proto} {`
can we add the resp version number for each test? 
it will show the resp when the test fail, but also will add more diff

Reply:
yeah, thought about it too, and was lazy...  i'll get it done.